### PR TITLE
39 enemies

### PR DESCRIPTION
**Pull request by chanson02**
Add enemy characters into the game.

### CODE REVIEW

File: dunwoody_disaster/views/MapScreen.py [hunk 1]
```diff
@@ -136,16 +139,17 @@ def from_json(json: dict, char: Character) -> "MapScreen":
...
-        ms.addRoom("English", (770, 366), test_enemy, "no_texture")
-        ms.addRoom("Science", (490, 217), test_enemy, "no_texture")
-        ms.addRoom("Dean's Office", (90, 589), test_enemy, "no_texture")
+        ms.addRoom("Bus Stop", (419, 700), chars.JoeAxberg(), "no_texture")
+        ms.addRoom("Court Yard", (693, 559), chars.LeAnnSimonson(), "no_texture")
+        ms.addRoom("Commons", (451, 449), chars.RyanRengo(), "no_texture")
+        ms.addRoom("Math", (236, 359), chars.NoureenSajid(), "no_texture")
+        ms.addRoom("English", (770, 366), chars.enemy1(), "no_texture")
+        ms.addRoom("Science", (490, 217), chars.enemy2(), "no_texture")
+        ms.addRoom("Dean's Office", (90, 589), chars.BillHudson(), "no_texture")
```
Comment (chanson02) [inline dunwoody_disaster/views/MapScreen.py]: @EngineeringDude123 would you be able to source images for these? I propose we use 1024x1024 character images. I can help with resizing if it's a bit off, but getting as close as possible to the target resolution is going to have the least impact on quality.

@EngineeringDude123 @MLHumphries @NoahBartell I need two other teachers to put in, any suggestions?

File: dunwoody_disaster/views/MapScreen.py [hunk 2]
```diff
@@ -26,12 +26,16 @@ def __init__(self, character: Character, entryPoint: Optional[tuple[int, int]]):
...
         self._callback = callback
 
     def keyPressEvent(self, event: QKeyEvent):
         if event.key() == Qt.Key_Enter or event.key() == Qt.Key_Return:
             if self.current_room:
-                self._callback()
+                self._callback(self.current_room)
 
     def init_ui(self):
         # layout = QGridLayout()
```
Comment (chanson02) [inline dunwoody_disaster/views/MapScreen.py]:
```suggestion
```

File: dunwoody_disaster/views/MapScreen.py (hunk 1 above)
Comment (MLHumphries) [reply dunwoody_disaster/views/MapScreen.py]: @chanson02 You could add Rob Bentz, Amalan Pulendran, Matthew Beckler, or if you want a real throwback John Anderson

Comment (chanson02) [reply dunwoody_disaster/views/MapScreen.py]: I never had classes with Rob or John, I'd be okay adding Amalan and Matthew though.

Comment (EngineeringDude123) [reply dunwoody_disaster/views/MapScreen.py:153]: Bill Hudson

Comment (EngineeringDude123) [reply dunwoody_disaster/views/MapScreen.py]: There's a teacher named David White who is basically a Ryan Rengo for the Cloud engineering students. We can add him as a DLC boss for my buddies in that program

**Review by EngineeringDude123 — APPROVED** [review]

**Review by MLHumphries — APPROVED** [review]

**Pull request merged**